src/parsing.jl

```diff
@@ -32,56 +35,48 @@ function argparse(cmd_args=ARGS)
         "link"
             action = :command
             help = "Path to an existing julia build you'd like to use with playgrounds."
-        "build"
-            action = :command
-            help = "The git url to clone the julia source from"
+        # "build"
+        #     action = :command
+        #     help = "The git url to clone the julia source from"
     end
 
     @add_arg_table parse_settings["create"] begin
         "dir"
             help = "Where to create the playground. Defaults to the current working directory (can be changed in `~/.playground/config`."
-            action = :store_arg
             default = ""
         "--requirements", "-r"
             help = "A REQUIRE or DECLARE file of dependencies to install into the playground."
-            action = :store_arg
             default = ""
         "--name", "-n"
             help = "A global name to allow activating the playground from anywhere."
-            action = :store_arg
             default = ""
         "--julia-version", "-j"
             help = "The version(s) of julia available to use. If multiple versions are provided the first entry will be the one used by `julia`. By default the user/system level version is used."
-            action = :store_arg
             default = ""
         "--req-type", "-t"
             help = "If --requirments isn't being passed a path ending in REQUIRE or DECLARE file, please specify which type is it \"REQUIRE\" or \"DECLARE\""
-            default = "REQUIRE"
+            arg_type = Symbol
+            default = :REQUIRE
```

Comment:
ArgParse just takes strings as inputs. The arg_type is just converting the string to the type given. Your example will work as intended. Note that you can see this behavior in the tests.